kukkuu/tests/conftest.py

```diff
@@ -1,8 +1,42 @@
+import secrets
+
 import pytest
 
 from kukkuu.service import get_hashid_service
+from kukkuu.tests.utils.jwt_utils import generate_symmetric_test_jwt
+from users.factories import UserFactory
 
 
 @pytest.fixture
 def hashids():
     return get_hashid_service()
+
+
+@pytest.fixture(autouse=True)
+def oidc_browser_test_api_token_auth_settings(settings):
+    settings.OIDC_BROWSER_TEST_API_TOKEN_AUTH = {
+        "ENABLED": True,
+        "AUDIENCE": ["kukkuu-api-dev", "profile-api-test", "kukkuu-admin-ui-test"],
+        "API_SCOPE_PREFIX": "",
+        "REQUIRE_API_SCOPE_FOR_AUTHENTICATION": False,
+        "API_AUTHORIZATION_FIELD": "authorization.permissions.scopes",
+        "ISSUER": "https://kukkuu-ui.test.hel.ninja",
+        "AUTH_SCHEME": "Bearer",
+        "USER_RESOLVER": "helusers.oidc.resolve_user",
+        "OIDC_CONFIG_EXPIRATION_TIME": 86400,
```

Comment:
What code uses these settings? Just curious because you haven't set them anywhere else in this PR at least.